src/Microsoft.SqlTools.ServiceLayer/TableDesigner/Constants.cs

```diff
@@ -11,6 +11,7 @@ public static class TablePropertyNames
         public const string Schema = "schema";
         public const string Description = "description";
         public const string Columns = "columns";
+        public const string ForeignKeys = "";
```

Comment:
any reason to use empty string?

Reply:
good catch. didn't notice this because we haven't enabled editing. :)